rel/overlay/etc/default.ini

```diff
@@ -232,8 +232,15 @@ socket_options = [{sndbuf, 262144}]
 [ssl]
 port = 6984
 
-; [chttpd_auth]
-; authentication_db = _users
+;[chttpd_auth]
+; These options are moved from [couch_httpd_auth]
+;authentication_db = _users
```

Comment:
We don't want to move the authentication_db. Notice that chttpd_auth already has its own authentication db entry.

Reply:
Changed back.